migrations/versions/rev-2020-11-12-09:02:31-aa9daed401f7_.py

```diff
@@ -0,0 +1,32 @@
+"""empty message
+
+Revision ID: aa9daed401f7
+Revises: 5e7082330d80
+Create Date: 2020-11-12 09:02:31.174541
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+
+# revision identifiers, used by Alembic.
+revision = 'aa9daed401f7'
+down_revision = '5e7082330d80'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column('speakers_calls', 'announcement',
```

Comment:
After this line, please add statement for converting empty string to null as well